### ISSUE TITLE
New e15 clipboard simulator doesn't handle well CRLF

### ISSUE THREAD

**Issue by RedGl0w** [numworks/epsilon#1796]
#### Describe the bug
When using windows for instance, and browsing on the web simulator, when we paste a string from the clipboard, the CR aren't removed (because epsilon is made for LF).

#### Screenshots
From @cent20 
![image](https://user-images.githubusercontent.com/43498612/110091759-17efec80-7d99-11eb-8818-97b007dab7f7.png)

#### Expected behavior
To remove CRLF from the clipboard

#### Environment
 - Epsilon 15
 - The platform(s) on which the problem happens: simulator (windows and web on windows)

**Comment by cent20net** [comment]
Bonjour,

Ce matin j'ai voulu tester la fonctionnalité "sauvegarde de l'état" et partage de celui-ci via un lien court 😍
C'est d'ailleurs mieux que ce que l'on avait demandé : 
https://github.com/numworks/epsilon/pull/1556
Merci à l'équipe pour ce développement intéressant.

J'ai donc pris un script : 
https://workshop.numworks.com/python/cent20/nsi4ever
Je l'ai copié collé dans l'émulateur web 
J'ai généré le lien court
https://workshop.numworks.com/simulators/1vy9pia
https://workshop.numworks.com/simulators/da3jn
Et il n'y avait pas mon script 😒

Merci à @RedGl0w pour la version EN de ce bug.

**Comment by RedGl0w** [comment]
> J'ai donc pris un script :
> https://workshop.numworks.com/python/cent20/nsi4ever
> Je l'ai copié collé dans l'émulateur web
> J'ai généré le lien court
> https://workshop.numworks.com/simulators/1vy9pia
> https://workshop.numworks.com/simulators/da3jn
> Et il n'y avait pas mon script unamused

Indeed, the bug you're reporting is totally different. The new system of recording session on the web simulator is a bit special : it record keypresses, and when you open the link given by the website, it "press" it instantly, so the paste event won't work when storing the session (that would need to change a bit how the new system works, and every copy / paste event, store in the `Module.ionEvents` with the event, the content of the clipboard).

**Comment by RedGl0w** [comment]
> 
> 
> Bonjour,
> 
> Ce matin j'ai voulu tester la fonctionnalité "sauvegarde de l'état" et partage de celui-ci via un lien court 😍
> C'est d'ailleurs mieux que ce que l'on avait demandé :
> #1556
> Merci à l'équipe pour ce développement intéressant.
> 
> J'ai donc pris un script :
> https://workshop.numworks.com/python/cent20/nsi4ever
> Je l'ai copié collé dans l'émulateur web
> J'ai généré le lien court
> https://workshop.numworks.com/simulators/1vy9pia
> https://workshop.numworks.com/simulators/da3jn
> Et il n'y avait pas mon script 😒
> 
> Merci à @RedGl0w pour la version EN de ce bug.

After having looked at how this new session system is done in epsilon, you should probably open an issue (because it is in epsilon :p). If you want I can do it for you

**Comment by cent20net** [comment]
> After having looked at how this new session system is done in epsilon, you should probably open an issue (because it is in epsilon :p). If you want I can do it for you

Voilà, on fait comme ça ! 😁
